README.md

```diff
@@ -13,10 +13,23 @@ Python 3.7.2
 1. To use Bridge, [download a release](https://github.com/urbit/bridge/releases) and unzip it (`bridge-$version.zip`).
 2. Open up your command line interface (Terminal on MacOS, Command Prompt on Windows).
 3. `cd` into the `bridge-$version` directory.
-4. Run this command: `python3 -m http.server 5000 --bind 127.0.0.1`
+4. Run this command: `python3 -m http.server 5000 --bind 127.0.0.1`. If you're using Bridge with a Ledger hardware wallet, skip this step and see the Ledger Support subsection below.
 5. You can then use Bridge by navigating to http://localhost:5000 using a web browser.
 
-### Verify checksums
+#### Ledger Support
```

Comment:
Since this is a more rare usage case than "verify checksums", I'd put it below that.

Also, this needs to be a h3, not h4, to match up with the other headers here.

Reply:
Disagree with this change. I had explicitly made it a subsection – running the software for Ledger is a special case of running the software. All instructions for running the software should be grouped together.

I debated whether verify checksums should actually come first. I'm open to that, but I also suspect that verifying checksums is something only people who already know how to do it, or are looking for it, are going to do.

Reply:
Oh, hmm, I suppose that's fair then.

If you can come up with nice copy for "do these steps instead of the above" for this Ledger section, by the way, then go for it. (As per your comment on #33.)

Reply:
Done – have a look